test/utils/test-utils.js

```diff
@@ -78,6 +84,15 @@ export async function deleteFixturesIfExist(modelName, whereClause) {
   return models[modelName].destroy({ where: whereClause || {} })
 }
 
+export async function deleteAllFixtures() {
+  for (const model of Object.keys(models)) {
```

Comment:
this whole thing could read clearer as:
```ts
await Promise.all(Object.keys(models).filter(m => m !== 'UserRole').map(deleteFixturesIfExist))
```

ymmv

Reply:
Trying to delete some models and their dependencies at the same time causes intermittent database deadlocks at least on my machine, so you need to delete each type of model consecutively.